files/podpicker/update.json

```diff
@@ -3,7 +3,6 @@
   "name": "PodPicker",
   "repo": "RoberMac/PodPicker",
   "files": {
-    "basePath": "dist/",
-    "exclude": ["PodPicker.min.js", "PodPicker.min.css"]
```

Comment:
replace "exclude" with "include". We only need the minified files, not everything